mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Kitto.Mixfile do
     [{:cowboy, "~> 1.0.0"},
      {:plug, "~> 1.3"},
      {:poison, "~> 3.0"},
-     {:fs, "~> 2.11.0"},
+     {:fs, github: "synrc/fs"},
```

Comment:
I did. I looked further into exfswatch and it just seems to be a layer on top of fs so we'd end up with the same issues.